.env.example

```diff
@@ -20,4 +20,8 @@ REACT_APP_USE_RECAPTCHA=
 REACT_APP_RECAPTCHA_SITE_KEY=
 
 # has to be inserted in the env file to use plugins and other websocket based features.
-REACT_APP_BACKEND_WEBSOCKET_URL=ws://localhost:4000/graphql
\ No newline at end of file
+REACT_APP_BACKEND_WEBSOCKET_URL=ws://localhost:4000/graphql
+
+# If you want to logs Compiletime and Runtime error , warning and info write YES or if u want to  
+# keep the console clean leave it blank
+LOGS_CONFIG_OVERRIDE=""
```

Comment:
"LOGS_CONFIG_OVERRIDE" is not self explanatory. please make it something easy to understand. Something like "SHOW_LOGS", "ALLOW_LOGS", etc.

Reply:
yes sir I also think that would be better and less confusing.